.gitignore

```diff
@@ -25,3 +25,11 @@ app/test/bad_block_*
 # do not include downloaded embedded binaries
 internal/embedding/**
 **/*token.sh
+# ignore talis files
+app.toml
+config.toml
+config.json
+scripts/vars.sh
+scripts/validator_init.sh
+scripts/upload_traces.sh
+scripts/user_scripts/start_tshark.sh
```

Comment:
do we want to just commit these?

Reply:
yeah, we want to be able to download them each time we call `talis init`. they become part of the payload and are responsible for starting the network. the reasoning for copying them each time is often I've wanted to change something for a specific experiment (ie turn on or off BBR etc)